### PR TITLE
Cookie initialy baked by NetworkToCode Cookie Drift Manager Tool

### PR DESCRIPTION
**Pull request by snaselj**
Template:

```
{
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "dir": "nautobot-app",
    "ref": "develop",
    "path": null
}
```

Cookie:

```
{
    "remote": "https://github.com/nautobot/nautobot-plugin-device-onboarding.git",
    "path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-device-onboarding",
    "repository_path": "/opt/ntc/drift-manager/outputs/nautobot-plugin-device-onboarding",
    "dir": "",
    "branch_prefix": "drift-manager",
    "context": {
        "codeowner_github_usernames": "@smith-ntc",
        "full_name": "Network to Code, LLC",
        "email": "info@networktocode.com",
        "github_org": "nautobot",
        "plugin_name": "nautobot_device_onboarding",
        "verbose_name": "Device Onboarding",
        "plugin_slug": "nautobot-device-onboarding",
        "project_slug": "nautobot-plugin-device-onboarding",
        "repo_url": "https://github.com/nautobot/nautobot-plugin-device-onboarding",
        "base_url": "nautobot-device-onboarding",
        "min_nautobot_version": "2.0.0",
        "max_nautobot_version": "2.9999",
        "camel_name": "NautobotDeviceOnboarding",
        "project_short_description": "Device Onboarding",
        "model_class_name": "None",
        "open_source_license": "Apache-2.0",
        "docs_base_url": "https://docs.nautobot.com",
        "docs_app_url": "https://docs.nautobot.com/projects/device-onboarding/en/latest",
        "_template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
        "_output_dir": "/opt/ntc/drift-manager/outputs",
        "_repo_dir": "/opt/ntc/drift-manager/outputs/.cookiecutters/cookiecutter-nautobot-app/nautobot-app",
        "_checkout": "develop"
    },
    "base_branch": "develop",
    "remote_name": "origin",
    "pull_request_strategy": "PullRequestStrategy.CREATE",
    "post_actions": [
        "PostAction.BLACK"
    ],
    "baked_commit_ref": "",
    "draft": true
}
```

CLI Arguments:

```
{
    "cookie_dir": "",
    "input": false,
    "json_filename": "setup-cookie-device-onboarding.json",
    "output_dir": "./outputs",
    "push": true,
    "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
    "template_dir": "nautobot-app",
    "template_ref": "develop",
    "pull_request": null,
    "post_action": [
        "black"
    ],
    "disable_post_actions": false,
    "draft": true
}
```

### CODE REVIEW

**Comment by snaselj** [comment]
This PR is ready for review.

cc.: @whitej6

File: development/docker-compose.base.yml [hunk 1]
```diff
@@ -22,13 +22,13 @@ services:
...
         condition: "service_healthy"
     <<:
-      - *nautobot-build
       - *nautobot-base
+      - *nautobot-build
   worker:
     entrypoint:
       - "sh"
       - "-c"  # this is to evaluate the $NAUTOBOT_LOG_LEVEL from the env
-      - "watchmedo auto-restart --directory './' --pattern '*.py' --recursive -- nautobot-server celery worker -l $$NAUTOBOT_LOG_LEVEL --events"  ## $$ because of docker-compose
```
Comment (whitej6) [inline development/docker-compose.base.yml:31]: Needs to be reverted

Comment (whitej6) [reply development/docker-compose.base.yml:31]: NVM covered by dev

File: docs/dev/arch_decision.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Architecture Decision Records
+
+The intention is to document deviations from a standard Model View Controller (MVC) design.
+
+!!! warning "Developer Note - Remove Me!"
+    Optional page, remove if not applicable.
+    For examples see [Golden Config](https://github.com/nautobot/nautobot-plugin-golden-config/tree/develop/docs/dev/dev_adr.md) and [nautobot-plugin-reservation](https://github.com/networktocode/nautobot-plugin-reservation/blob/develop/docs/dev/dev_adr.md).
```
Comment (whitej6) [inline docs/dev/arch_decision.md:7]: Either delete or needs information

**Review by whitej6 — CHANGES_REQUESTED** [review]

File: docs/dev/arch_decision.md (hunk 2 above)
Comment (snaselj) [reply docs/dev/arch_decision.md:7]: Should be done by package maintainers @whitej6

Comment (snaselj) [reply docs/dev/arch_decision.md]: File has been removed.

Comment (scetron) [reply docs/dev/arch_decision.md]: Done.

**Review by whitej6 — APPROVED** [review]

**Review by scetron — APPROVED** [review]

**Pull request merged**